analyses/molecular-subtyping-EWS/01-run-subtyping-ewings.Rmd

```diff
@@ -0,0 +1,120 @@
+---
+title: "Add molecular subtype for samples that have hallmark Ewings Sarcoma fusions"
+output: html_notebook
+author: K S Gaonkar for D3b
+date: January 2020
+---
+
+Identify sample IDs with hallmark _EWSR1_ fusions and subtype as `EWS` 
```

Comment:
Can you add a bit more specifics in this preview here? Things like, what is your input and what is your intended output.

Reply:
Also maybe a sentence about your strategy.